build_tools/travis/before_install.sh

```diff
@@ -28,6 +28,10 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
         brew install vtk
     fi
 
+    if [[ "$USE_OMP" == 1 ]]; then
+        brew install llvm
+    fi
+
```

Comment:
@agramfort  please is that the correct formula we need to build with openMP ? or clang support it by default ?